website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -2,79 +2,95 @@
-### Installing the Chart
+</TabItem>
+<TabItem label="Google Cloud Kubernetes" value="GoogleCloudKubernetes">
 
-1. Add Appsmith into your repository using Helm.
+  1. [Configure cluster access for kubectl](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl) on Google Cloud Kubernetes.
+  2. Ensure that you have a default storage class running. If not, [set up a default storage class on Google Kubernetes Engine](https://cloud.google.com/anthos/clusters/docs/on-prem/1.3/how-to/default-storage-class)[GKE].
+  3. The NGINX Ingress Controller should be enabled by default. If not, [install ingress controller on Google Kubernetes Engine](https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke)[GKE]. Ensure that you enable the correct version for your cluster.
 
+</TabItem>
+<TabItem label="AWS EKS" value="AWSEKS">
+
+   1. Set up the Kubernetes cluster by [creating a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
+    2. Ensure that you have a default storage class running. If not, [create default storage class](https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html)
+    3. The NGINX Ingress Controller should be enabled by default. If not, [install NGINX Controller for AWS EKS](https://kubernetes.github.io/ingress-nginx/deploy/#network-load-balancer-nlb).
+    Ensure that you enable the correct version for your cluster.
+
+</TabItem>
+<TabItem label="Microk8s" value="Microk8s">
+
+  1. Set up Kubernetes by following [working with kubectl](https://microk8s.io/docs/working-with-kubectl).
+  2. Ensure that you have a default storage class running. If not, [enable storage class](https://microk8s.io/docs/command-reference#heading--microk8s-enable).
+  3. The NGINX Ingress Controller should be enabled by default. If not, [add on: Ingress](https://microk8s.io/docs/addon-ingress). Ensure that you enable the correct version for your cluster.
+    
+</TabItem>
+</Tabs>
+
+## Install
+
+1. Click to download the [`values.yaml`](https://bit.ly/3ETEgPT) file, and place it into the Appsmith installation folder.
+
+OR, run the following cURL:
+
+```bash
+curl -L https://bit.ly/3ETEgPT -o $PWD/values.yml
 ```
+
+2. Add Appsmith into your repository using Helm.
+
+```bash
 helm repo add appsmith https://helm.appsmith.com
 
 helm repo update
 ```
 
-2\. Install the chart with the <mark>`appsmith`</mark> release.
+2. Install the chart with the <mark>`appsmith`</mark> release.
 
-```
+```bash
 helm install appsmith/appsmith --generate-name
 ```
 
-The command deploys the Appsmith application on the Kubernetes cluster in the default configuration. The [Parameters](https://github.com/appsmithorg/appsmith/tree/release/deploy/helm#paramters) section lists the configurable parameters during installation.
+The command deploys Appsmith on the Kubernetes cluster with the default configuration. Read more on [configurations](#configuration) that you could do during installation.
```

Comment:
Do we really need to cross-reference the Configuration section, which is right below the Install on the same page? You can maybe rewrite this as

> The command deploys the Appsmith application on the Kubernetes cluster with the default configuration. You can configure the chart using the parameters shown in the following section.

Reply:
Yes, we can remove this.